sprouts.py

```diff
@@ -0,0 +1,372 @@
+from math import sqrt
+import sys
+import time
+import random
+import pygame
+import pygame_menu
+from pygame import *
+
+
+#Contains Game Settings: sproutset.py on Github
+from sproutset import Settings
+
+####OBJECTS 
+#Game Window 
+pygame.init()
+ss = Settings()
+win = pygame.display.set_mode((ss.width, ss.height))
+pygame.display.set_caption('The Sprouts Game')
+
```

Comment:
I would maybe add more documentation about what you use these objects for specifically. Also later in the code, there could be a bit more explaining :)

Reply:
Good to know! It should run if you have both: pygame and pygame_menu (which require actual installation - pip(3) install on Mac, not 100% for windows). Also having sproutset and the arialt.tff file within the same directory is all it needs.